### ISSUE TITLE
install on server problem

### ISSUE THREAD

**Issue by xtoolkit** [Microsoft/WSL#3145]
* Your Windows build number:  Microsoft Windows [Version 10.0.17134.1]

hi, i install wsl (ubunbu) on server with this guide: [install-on-server](https://docs.microsoft.com/en-us/windows/wsl/install-on-server)
wsl good work, but when shutdown and turn on, wsl has error:
```
Windows Subsystem for Linux has no installed distributions.
Distributions can be installed by visiting the Microsoft Store:
https://aka.ms/wslstore
Press any key to continue...
```
and i have to install again wsl and install my dpkg ...
Is there a solution?
sry 4 my bad english

**Comment by jaredlt** [comment]
For your reference, I got this this morning when I turned my computer on. AFAIK there was no Windows update so I was a bit perplexed as to what had caused it.

On the advice of this comment https://www.reddit.com/r/bashonubuntuonwindows/comments/8j5igr/subsystem_using_lxss_folder_broke_today/dyx9ubu/ I just rebooted my computer (literally did nothing else) and that fixed it.

Still have no idea what caused it.

Win 10 Version	10.0.17134 Build 17134
